data/capabilities/vulnerability-data.yaml

```diff
@@ -285,9 +283,6 @@ sources:
     name: Microsoft AzureLinux OVAL
     url: https://github.com/microsoft/AzureLinuxVulnerabilityData/blob/main/azurelinux-3.0-oval.xml
     vunnel_provider: mariner
-    advisories:
-      - name: AzureLinux Security Advisory
```

Comment:
I think another issue here is that Azure Linux is the real name for Mariner Linux, but they're still listed separately. I wasn't sure how to fix that.